### DISCUSSION TITLE
Claim to bedrock

### DISCUSSION THREAD

**Discussion by SakuraGoose** [GriefPrevention/GriefPrevention#1790] [Q&A]

**Comment by Jikoo** [comment]
Download the most recent dev build off the CI and set `MaximumDepth` to bedrock level or below in your configuration.

**Comment by L3v3lup** [comment]
I do have the same problem. That did not work for me. (Using 16.18-RC1)

**Reply by Jikoo** [reply]
16.18-RC1 has a known issue with automatic extension of existing claims. Try in a new claim.

0e3a2e9791, which fixes the issue, is no longer available via the CI because a month has elapsed since it was committed. You can download it off JitPack instead. https://jitpack.io/com/github/TechFortress/GriefPrevention/0e3a2e9791/GriefPrevention-0e3a2e9791.jar

**Comment by L3v3lup** [comment]
Thanks, but that didn't fix it in my case.
I have it set MaximumDepth to -64. Instead of going to 0 it now ends at (positive) 45. So it is even worse now.
It's a new claim too.

**Reply by Jikoo** [reply]
And you placed blocks and the claim did not extend down? By default claims only extend down so far until they have a reason to expand, usually block placement.

**Reply by L3v3lup** [reply]
> And you placed blocks and the claim did not extend down? By default claims only extend down so far until they have a reason to expand, usually block placement.

After playing around for some time, breaking and placing more blocks, the claim finally extended down to -64.
It seems to work... more or less, i guess. But thats good enough for me :) Thank you

**Reply by TheGorshWithin** [reply]
I also have this issue! is there currently some type of resolution for this problem?
